ParkingZoneApp/Areas/Admin/Views/ParkingSlot/Index.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@model IEnumerable<ParkingZoneApp.ViewModels.ParkingSlotViewModels.ListOfParkingSlotsVM>
+
+@{
+    ViewData["Title"] = "Index";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h1>Index</h1>
+
+<p>
+    <a asp-controller="ParkingSlot" asp-action="Create" class="btn btn-primary">Create New</a>
```

Comment:
Where is Back button? After entering into Slots you should be able to go back to ParkingZone Index.